docs/README.md

```diff
@@ -9,11 +9,13 @@
   - [Start the application server](#start-the-application-server)
   - [Stop the application server](#stop-the-application-server)
   - [Destroy all the things](#destroy-all-the-things)
-- [Working with OAuth](#working-with-oauth)
-- [Frontend - Working with CSS and JS](#frontend---working-with-css-and-js)
+- [Working with OAuth](#oauth)
+- [Frontend - Working with CSS and JS](#frontend)
+- [Localization and Internationalization](#localization-and-internationalization)
```

Comment:
:thinking: Where is this coming from?

Reply:
from `doctoc`, it seems like we didin't run it in this file for a while

Reply:
I see... But I guess the link to the `href="localization-and-internationalization" should be added as well in the section?

Reply:
Ah no, in github it works like that by convention, the reason the other's have links is because (at least at the beginning) some of them were shorter than the auto-generated ones.